database_mongo/seed.js

```diff
@@ -191,26 +194,37 @@ const createLinks = () => {
   return linksArr;
 };
 
-const productPlatforms = assignPlatforms();
-const productLinks = createLinks();
-const productSysReq = createSystemRequirements();
-const productSteamRate = createSteamRatings();
-const productDevelopers = createDevelopers();
-const productPublishers = createPublishers();
+
+//seed script
+
+var recordCounter = 1;
 
 const seed = () => {
+
+  //generators
+  const productPlatforms = assignPlatforms();
+  const productLinks = createLinks();
+  const productSysReq = createSystemRequirements();
+  const productSteamRate = createSteamRatings();
+  const productDevelopers = createDevelopers();
+  const productPublishers = createPublishers();
+
+
+  //docs builder
   const docsArray = [];
 
-  for (let i = 0; i < 100; i++) {
+  for (let i = 0; i < recordsNum; i++) {
     const newDoc = {};
 
     newDoc.steam_rating = null;
-    newDoc.product_id = i + 1;
+    newDoc.product_id = recordCounter;
```

Comment:
Have you tried generating 10M records with this algo? Looks like the counter for your id primary key resets to 1 every time the function is called, and with 10,000 records as a batch size, that might result in some id conflicts (for SQL, anyway)

Reply:
(Also note that since you're using Postgres, it's better to not insert a primary key at all, since it doesn't update the auto-increment value, resulting in errors further down the line when attempting to make a POST request.) I found this out the hard way!